setuptools/independent/DepsProcessor/project.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "NuGet.RuntimeModel": "3.5.0-rc-1285",
     "NuGet.Versioning": "3.5.0-rc-1285",
-    "System.CommandLine": "0.1.0-e160119-1",
+    "System.CommandLine": "0.1.0-e170322-6",
```

Comment:
My concern is that we might end up polluting our official core-setup builds from packages on that feed without realizing it. That feed is for experiential things and we need to be certain we don't take a shipping dependency on any of those.